### PR TITLE
Wrote a script that creates SNS topics for each base genre.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

**Review by Leon-JS02 — DISMISSED** [review]
Excited to see what the email looks like!

File: load_to_rds/create_topic_message.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    return message
+
+
+if __name__ == "__main__":
+
+    with open("gog_data.json", "r") as f:
+        games = load(f)
+
+    with open("message_html.html", "w") as f:
+        f.write(create_html(games["listings"][2:], "Cool"))
```
Comment (paulauus) [inline load_to_rds/create_topic_message.py:57]: What's cool? :)

**Review by paulauus — DISMISSED** [review]

File: load_to_rds/create_topic_message.py (hunk 1 above)
Comment (mehrdadhalali) [reply load_to_rds/create_topic_message.py:57]: It was a test lol

File: load_to_rds/create_ugly_topic_message.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return message
+
+
+if __name__ == "__main__":
+
+    with open("gog_data.json", "r") as f:
+
+        games = load(f)
+
+    with open("ugly_msg.txt", "w") as f:
```
Comment (Leon-JS02) [inline load_to_rds/create_ugly_topic_message.py:27]: 👁️

File: load_to_rds/create_ugly_topic_message.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+"""Turns out SNS doesn't support HTML, this is a placeholder script to just format the messages in a readable way."""
+
+from json import load
+
+from create_topic_message import format_price
+
+
+def create_ugly_disgusting_message_i_hate_it(games: list[dict], genre: str):
```
Comment (Leon-JS02) [inline load_to_rds/create_ugly_topic_message.py:8]: ...

**Review by Leon-JS02 — APPROVED** [review]
A good temporary workaround.

**Review by MichaelHassan — APPROVED** [review]
Nice changes, good small functions.

**Pull request merged**